R/dpconf_get.R

```diff
@@ -67,27 +67,32 @@ dpconf_write <- function(project_path, dpconf){
 dpconf_validate <- function(dpconf, project_path){
   #TODO: add additional validation for repo test as well as data remote test
   # specific to class of config
-  
+
   if(nchar(Sys.getenv("GITHUB_PAT")) == 0)
-    stop(cli::format_error(glue::glue("Could not get dpconf as it did not find", 
+    stop(cli::format_error(glue::glue("Could not get dpconf as it did not find",
                                       " GITHUB_PAT in the environment. Set it ",
                                       "by Sys.setenv(GITHUB_PAT = <your github",
                                       " personal access token>) and retry!")))
-  
+
   repo <- git2r::repository(path = project_path)
   repo_url <- git2r::remote_url(repo = repo)
-  
-  if(!is.null(repo_url)){
+
+  if(!is.null(repo_url)){ # TODO do something if repo url not found here?
+    remote_alias <- git2r::remotes(repo = repo)
+
+    if(length(remote_alias) > 1){
+      stop("More than one remote alias found, delete one and try again")
```

Comment:
This is good, but for consistency of the look and feel of the errors, we should probably use `cli::format_error`